aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -663,9 +695,18 @@ public static ResourceModel.ResourceModelBuilder translateDbInstanceFromSdkBuild
 
         String domain = null;
         String domainIAMRoleName = null;
+        String domainAuthSecretArn = null;
+        String domainFqdn = null;
+        String domainOu = null;
+        List<String> domainDnsIps = null;
         if (CollectionUtils.isNotEmpty(dbInstance.domainMemberships())) {
-            domain = dbInstance.domainMemberships().get(0).domain();
-            domainIAMRoleName = dbInstance.domainMemberships().get(0).iamRoleName();
+            DomainMembership domainMembership = dbInstance.domainMemberships().get(0);
```

Comment:
If this is not a CMAD, but AWS managed domain - will those properties be properly populated?